specification/ml/_types/Job.ts

```diff
@@ -44,50 +45,50 @@ export class JobStatistics {
 }
 
 export class Job {
-  allow_lazy_open: boolean
+  allow_lazy_open?: boolean
   analysis_config: AnalysisConfig
   analysis_limits?: AnalysisLimits
   background_persist_interval?: Time
-  blocked?: JobBlocked
-  create_time?: integer
   custom_settings?: CustomSettings
   daily_model_snapshot_retention_after_days?: long
   data_description: DataDescription
-  datafeed_config?: Datafeed
-  deleting?: boolean
+  datafeed_config?: DatafeedConfig
   description?: string
-  finished_time?: integer
   groups?: string[]
-  job_id: Id
+  job_id?: Id
   job_type?: string
-  job_version?: VersionString
   model_plot_config?: ModelPlotConfig
-  model_snapshot_id?: Id
-  model_snapshot_retention_days: long
+  model_snapshot_retention_days?: long
   renormalization_window_days?: long
-  results_index_name: IndexName
+  results_index_name?: IndexName
   results_retention_days?: long
 }
-
-export class JobConfig {
-  allow_lazy_open?: boolean
+export class JobRead implements OverloadOf<Job> {
+  allow_lazy_open: boolean
   analysis_config: AnalysisConfig
   analysis_limits?: AnalysisLimits
   background_persist_interval?: Time
+  blocked?: JobBlocked
+  create_time?: integer
   custom_settings?: CustomSettings
   daily_model_snapshot_retention_after_days?: long
   data_description: DataDescription
-  datafeed_config?: DatafeedConfig
+  datafeed_config?: DatafeedConfig /** TODO: Make DatafeedConfigRead */
```

Comment:
I can't use `DatafeedConfigRead` here as must match `implemented OverloadOf<...>` type for the field. Can we loosen this requirement to allow for changing the type?